### DISCUSSION TITLE
[FR] add allow and blocklist support

### DISCUSSION THREAD

**Discussion by ComfortablyCoding** [ComfortablyCoding/strapi-plugin-transformer#37] [Ideas]
Add support for adding allow and block lists for certain routes.

Most likely will either be an array or function that is passed the current route as the parameter and must return a boolean indicating if the route should be processed.

Example
```js
// allow any routes that have foo anywhere in the route string
allowList: (route) => new RegExp('foo').test(route);
```

#### Linked Issue
#33

**Comment by BabyDino** [comment]
Running into this issue now. I am writing a lot of custom resolvers and like to be in complete control of the output without using the transformer.

**Reply by ComfortablyCoding** [reply]
Will see if I can get this implemented sometime this weekend. In the meantime you can use the `Strapi-Transformer-Ignore` header as true as a temporary solution to achieve this on a per request basis.

**Comment by ComfortablyCoding** [comment]
Added in v3 via #64
